.github/workflows/main.yml

```
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.6', '2.7', '3.0']
+        ruby-version: ['2.7']
```

I imagine if we are modifying this we want to cover all the latest ones? We run 3.1 so it would be off to not be testing those?

```suggestion
        ruby-version: ['2.7', '3.0', '3.1', '3.2', '3.3']
```

3.4 is landing soon as well

---

Yeah, sorry I didn't mean to only leave 2.7 before, so much as just drop the one that was busted. May as well keep the ones passing in for long term compatibility and maybe back merge